### PR TITLE
Remove the Java bits from this repository.

### PR DESCRIPTION
**Pull request by pphaneuf**
The Java parts of this has been moved to their own repository here:

https://github.com/google/certificate-transparency-java

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,8 +68,7 @@ Code Layout
 -----------
 
 The source code is generally arranged according to implementation language, in
-the `cpp`, `go`, `java` and `python` subdirectories.  The key subdirectories
```
Comment (codingllama) [inline README.md]: I think we should point people to the Java repo somewhere in this README, just in case.

Comment (pphaneuf) [inline README.md]: How about this?

File: README.md [hunk 2]
```diff
@@ -68,7 +68,10 @@ Code Layout
 -----------
 
 The source code is generally arranged according to implementation language, in
-the `cpp`, `go`, and `python` subdirectories.  The key subdirectories are:
+the `cpp`, `go`, and `python` subdirectories (Java is in
+a
```
Comment (codingllama) [inline README.md]: nit: move to the line above?

File: README.md (hunk 1 above)
Comment (codingllama) [inline README.md]: LGTM. Feel free to merge after you address the last comment.

File: README.md (hunk 2 above)
Comment (pphaneuf) [inline README.md]: Done.

**Pull request merged**